tests/run.sh

```diff
@@ -48,3 +48,6 @@ shakedown GET /browse/www.test.com/blah/blah
 shakedown GET /health
   status 200
   contains 'OK'
+
+shakedown GET /extension/blah
```

Comment:
Why blah?

Reply:
No particular reason. I copied the test for `user` links which also has "blah".  They could be replaced with realistic values, but then there is the fact that those values are not validated, and putting realistic values might confuse a reader into thinking that they are.

Reply:
Sounds great!